src/commands/apps-create.ts

```diff
@@ -0,0 +1,210 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+
+import * as Command from "../command";
+import * as getProjectId from "../getProjectId";
+import * as FirebaseError from "../error";
+import { AppPlatform, createAndroidApp, createIosApp, createWebApp } from "../appsCreate";
+import { prompt, promptOnce, Question } from "../prompt";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+
+function promptDisplayName(): Question {
+  return {
+    type: "input",
+    name: "displayName",
+    default: "",
+    message: "What would you like to call your app? ",
+  };
+}
+
+function logPostAppCreationInformation(
+  appMetadata: {
```

Comment:
this type is rather long. can this be pulled out into an interface?

Reply:
Yes, my intention is to use the [AppMetadata](https://github.com/firebase/firebase-tools/pull/1470/files#diff-42e483efc7f3fdb0f79fc306b7f264fbR16) types see (https://github.com/firebase/firebase-tools/pull/1470)

Reply:
Can you make that update now that #1470 is merged? Same for the similar cases that Bryan commented on below.